Makefile

```diff
@@ -93,12 +93,15 @@ bench-test:
 ##@ Continuous Integration
 
 ci: ## Run recipes for CI.
-ci: fmt clippy test bench-test check-cargotoml check-dirty-rpc-doc security-audit
+ci: fmt clippy test bench-test check-cargotoml check-whitespaces check-dirty-rpc-doc security-audit
 	git diff --exit-code Cargo.lock
 
 check-cargotoml:
 	./devtools/ci/check-cargotoml.sh
 
+check-whitespaces:
+	git diff-index --check --cached $$(git rev-parse --verify master 2>/dev/null || echo "4b825dc642cb6eb9a060e54bf8d69288fbee4904") --
+
```

Comment:
what's this `4b825dc642cb6eb9a060e54bf8d69288fbee4904`?

Reply:
Git genesis commit